src/desktop/apps/order2/Components/App.tsx

```diff
@@ -0,0 +1,24 @@
+import React, { Component } from 'react'
+import styled from 'styled-components'
+import { Layout } from './Layout'
+
+interface Props {}
+
+export class App extends Component<Props> {
+  componentDidMount() {
+    console.log('mounted on client')
```

Comment:
Stray log here - probably can kill the whole `componentDidMount`, right? And if so, then this could be a stateless component, no?

Reply:
This was just to make sure that client-side code was mounting and properly rendering after SSR; will be removed in subsequent PRs